styx-common/src/main/java/com/spotify/styx/monitoring/MetricsStats.java

```diff
@@ -148,6 +148,13 @@ public boolean isInstance(Metric metric) {
       .tagged("what", "workflow-consumer-error-rate")
       .tagged("unit", "error");
 
+  static final MetricId PUBLISHING_RATE = BASE
+      .tagged("what", "publishing-rate");
+
+  static final MetricId PUBLISHING_ERROR_RATE = BASE
+      .tagged("what", "publishing-error-rate")
+      .tagged("unit", "error");
```

Comment:
BTW I guess it's a wrong unit for a rate.

Reply:
I think we once had a PR to remove all those faulty units. No idea why they make their way back. :(